src/renderer/gl_engine/tvgGlTessellator.cpp

```diff
@@ -1722,11 +1722,22 @@ void Stroker::strokeCap()
 
 void Stroker::strokeLineTo(const Point& curr)
 {
+    // special case for line with zero length
```

Comment:
```
    // special case for line with zero length
    if (mStrokeState.prevPt == curr) {
        if (mStrokeCap == StrokeCap::Butt && tvg::zero(mStrokeWidth)) return;
        if (mStrokeCap == StrokeCap::Square) strokeSquarePoint(curr);
        else if (mStrokeCap == StrokeCap::Round) strokeRoundPoint(curr);
    }
```

Reply:
Thank you for suggesting a more concise form. Regarding the additional ```strokeDot``` method, wouldn't it be better to remove it? I have already introduced two additional methods for generating vertices, and we are not using complex conditions when triggering them.